src/bindings/python/tests_compatibility/test_onnx/test_backend.py

```diff
@@ -526,9 +526,6 @@ def expect_fail(test_case_path, xfail):  # type: (str) -> None
         "OnnxBackendNodeModelTest.test_bitwise_and_i16_3d_cpu",
         "OnnxBackendNodeModelTest.test_bitwise_and_i32_2d_cpu",
         "OnnxBackendNodeModelTest.test_bitwise_and_ui64_bcast_3v1d_cpu",
-        "OnnxBackendNodeModelTest.test_bitwise_not_2d_cpu",
-        "OnnxBackendNodeModelTest.test_bitwise_not_3d_cpu",
-        "OnnxBackendNodeModelTest.test_bitwise_not_4d_cpu",
```

Comment:
There are two similar files for the new and compatibility API tests. 
Could you please make the same change also in the `src/frontends/onnx/tests/tests_python/test_backend.py` file:
https://github.com/openvinotoolkit/openvino/blob/f3fc5032880aac72565466ca8e207bb211398685/src/frontends/onnx/tests/tests_python/test_backend.py#L397-L399

Reply:
@mitruska  Thanks for the review, removed those in the latest commit.